src/coreclr/tools/r2rtest/Crossgen2Runner.cs

```diff
@@ -65,6 +65,13 @@ public override ProcessParameters CompilationProcess(string outputFileName, IEnu
         {
             ProcessParameters processParameters = base.CompilationProcess(outputFileName, inputAssemblyFileNames);
             processParameters.Arguments = $"{Crossgen2Path} {processParameters.Arguments}";
+            // DOTNET_ variables
+            processParameters.EnvironmentOverrides["DOTNET_GCStress"] = "";
```

Comment:
Same question. If this is for .NET 6+, then only `DOTNET_` is needed.

Reply:
https://github.com/dotnet/runtime/pull/76997#discussion_r995150098